frotz/src/interface/frotz_interface.c

```diff
@@ -1756,21 +2030,7 @@ int filter_candidate_actions(char *candidate_actions, char *valid_actions, zword
     act_newline = malloc(strlen(act) + 2);
     strcpy(act_newline, act);
     strcat(act_newline, "\n");
-
-    // Step: Code is copied due to inexplicable segfault when calling step() directly; Ugh!
```

Comment:
@mhauskn I figured out the segfault. There's a `step` function in `regexp.c` (see https://code.woboq.org/userspace/glibc/misc/regexp.c.html#50). It was clashing with the one in Jericho, so I renamed it.